### ISSUE TITLE
set posix permissions in metadata 

### ISSUE THREAD

**Issue by dirkpetersen** [FredHutch/lf-backup#8]
at the end of the backup_file function put in custom meta data posix_permissions and store permissions of the file and parent folder with some delimiter: 
user,group,access,folderuser,foldergroup,folderaccess, for example:
55000,10100001,0664,0,10100002,2770

**Comment by bmcgough** [comment]
Do you need any metadata from the DB for this, or will lb-backup store this all with the file in the object store?

**Comment by dirkpetersen** [comment]
No, I think Jeff can grab all that info from the file system as he is backing up file by file
